app/helpers/image_service_helper.rb

```diff
@@ -82,13 +84,19 @@ def member_image_tag(parent_id:, member:, size_key: nil, lazy: false, use_image_
 
   private
 
-  def create_riiif_url(path)
-    if CHF::Env.lookup(:public_riiif_url)
-      url = Addressable::URI.parse(CHF::Env.lookup(:public_riiif_url))
-      raise "public_riiif_url requires a valid URL with host, eg `http://host` or `//host`" if url.host.nil?
-      return Addressable::URI.join(url, path).to_s
-    else
-      return path
+  attr_accessor :public_host
+
+  def public_host
+    @public_host ||= begin
+      url = Addressable::URI.parse(CHF::Env.lookup(:iiif_public_url))
```

Comment:
setting instance variable in a helper will set it on controller/viewcontext. If we need to do it, probably need to give it a less likely to conflict name like `@_image_service_helper_public_url`.  Or extract some of this to a helper class.